demos/text-field.html

```diff
@@ -205,10 +204,6 @@ <h2>Outlined Text Field</h2>
               }
               tf.layout();
             });
-            document.getElementById('outlined-custom-colors').addEventListener('change', function(evt) {
```

Comment:
You removed this code but it looks like you didn't add the equivalent class to use your new functions, so the "alternate colors" checkbox under outlined text field currently doesn't seem to do anything.

Moreover, should it also be possible to remove the outlined-dense and outlined-disable handlers here and use your centralized code for that, and ditto for the disabled, dense, and alternate colors checkboxes for the first example on the page?